imagely.bin.js

```diff
@@ -17,6 +22,41 @@ options.bg = args.bg || args.b;
 options.json = args.json || args.d;
 options.log = args.log || args.l;
 options.batch = args.batch;
+options.logFilepath = args.logFilepath;
+
+function addToBatchLogs(json) {
+	var log;
+	var dimensions;
+	var imageKey = json[0].key;
+	var imageKeyArray = imageKey.split('_');
+
+	try {
+		dimensions = imageSize(destination);
+	}
+	catch (exception) {
+		dimensions = { width: null, height: null };
+	}
+
+	log = dimensions;
+	log['uuid'] = imageKeyArray[1];
+	log['endpoint'] = imageKeyArray[0];
+
+	// Check if truthy; account for null, 0, or undefined.
+	if (log.width && log.height) {
+		logs.success.push(log);
+	}
+	else {
+		logs.failure.push(log);
+	}
+}
+
+function writeLogFile(logFilepath) {
+	fs.writeFile(logFilepath, JSON.stringify(logs, 0, 4), function(err) {
```

Comment:
The parameters passed to `JSON.stringify()` don't look correct: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify